packages/react-components/src/components/data-providers/tsp-data-provider.ts

```diff
@@ -271,6 +271,15 @@ export class TspDataProvider {
         return tooltipResponse.getModel()?.model;
     }
 
+    async fetchTreeTooltip(entryId: number): Promise<{ [key: string]: string } | undefined> {
+        // use start of state for fetching tooltip since hover time is not available
+        const time = BigInt(1);
```

Comment:
Hardcoded time set to 1? Does not match the comment above. If we are to allow fetching a tooltip for a tree element (not time related) then perhaps the TSP should be updated to make the requested_time optional. Also, if I understand correctly the TSP only supports fetching tooltip for a time graph?

Reply:
method will be removed